lib/preprocessor.js

```diff
@@ -60,6 +62,15 @@ var createCoveragePreprocessor = function(logger, basePath, reporters, coverageR
       // remember the actual immediate instrumented JS for given original path
       sourceCache[jsPath] = content;
 
+      if (includeAllSources) {
```

Comment:
@mwinche can you explain what does this code?

In preprocessor we pass file object, then we process content of the file using `instrument` function (which wrap source code to special functions). Why we and what we save in stubs?

Also what does mean `stubs`?

Thanks